### DISCUSSION TITLE
Graphical interface

### DISCUSSION THREAD

**Discussion by tommyblue** [tommyblue/smugmug-backup#55] [Ideas]
What do you think of giving this tool a graphical interface?
Which features do you think would be the most useful?

**Comment by Liguria360** [comment]
A basic graphical interface to build the config file would be fine for non-coding greenhorns on windows OS.
Just found your application trying to get it working for downloading ~20.000 files from my SmugMug account.
BTW, greetings from Chiavari to Firenze!!!

**Comment by tommyblue** [comment]
Hi @Liguria360 a very basical interface, in fact, already exists in the [electron](https://github.com/tommyblue/smugmug-backup/tree/electron) branch but it was a very experimental attempt to give a GUI to the application. Given your request, I could modify it to only allow the users to create their configuration file. In the meanwhile, you can give it a try but it requires some dev skills

**Comment by Liguria360** [comment]
Cioa Tommaso,
non era difficile di editare il file config.tml - fatto il backup con  [smugmug-backup_1.6.0_windows_amd64.tar.gz](https://github.com/tommyblue/smugmug-backup/releases/download/v1.6.0/smugmug-backup_1.6.0_windows_amd64.tar.gz)
Vado su jAlbum, SmugMug è diventato troppo caro per me.

Thomas Krüger | Chiavari (GE), Italia

Tommaso Visconti ***@***.***> schrieb am Dienstag, 17. Dezember 2024 um 23:02:

**Comment by Mumin67** [comment]
Hi Tommaso, and thank you for your contribution.  I would love a GUI. I love Smugmug but it is insane that we can´t do easy back up.

**Reply by tommyblue** [reply]
thanks @Mumin67 
I will try to create at least a first UI to create the configuration file

**Comment by Geekerbyname** [comment]
Hi all, just found this app, and it's going to work great for me, hopefully. 

If you're creating a GUI, it would be wonderful to have one that could set configs then run the backup, or even schedule it. All in a Docker container :)
